enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,466 @@
+4. If the placement has no old placementdicisions yet, create placementdicisions;
+5. Otherwise, update the existing placementdicisions, and add/remove placementdicisions if necessary.
+
+#### Workflow 2: Evaluate a cluster predicate
+1. If `ClusterName` is specified, return cluster if it is exists;
+2. If `ClusterSets` is not empty, get clusters belongs to intersection of `ClusterSets` and the clustersets bound to the placement namespace;
+3. Otherwise, get clusters from the clustersets bound to the placement namespace;
+4. If `RequiredDuringSchedulingRequiredDuringExecution` is specified, filter clusters with label/claim selectors; 
+5. Enforce the afinity constrains on clusters if exist;
+6. If `NumberOfClusters` is not specified, return all matched clusters;
+7. Otherwise, get the intersection of old matched clusters and new matched clusters;
+8. If `NumberOfClusters` is equal to the size of the intersection, return clusters in the intersection;
+9. if `NumberOfClusters` is less than the size of the intersection, select random clusters from the intersection and return;
+10. Otherwise, return clusters in the intersection with some random additional clusters from the remaining of the new matched clusters;
+
+**Note:** The old matched clusters are preferred during the predicate evaluation, which makes the placement decisions as stable as possible.
+
+### Graduation Criteria
+
+1. The new placement APIs starts with version v1alpha1;
+2. The version name will become v1beta1 once it gets stable and is able to cover all the capabilities of the current placementrule APIs; 
+3. Components depends on the current placementrule APIs can migrated to the new placement APIs (v1beta1) then. And a conversion webhook will be provided to migrated existing placement objects to the new placement APIs (v1beta1);
+4. The current placementrule APIs becomes deprecated and still exists in the next 3 releases.
+
+
+## Alternatives
+
+### 1. Setting clustersets explicitly in a `Placement` instead of binding clustersets to the placement namespace
+
+1. A normal user creates a placement with explicit references to clustersets;
+    ```yaml
+    apiVersion: cluster.open-cluster-management.io/v1alpha1
+    kind: Placement
+    metadata:
+      name: placement1
+      namespace: ns1
+    spec:
+      clusterSets:
+        - clusterset1
+        - clusterset2
+      predicates:
+        - labelSelector:
+            matchLabels:
+              vendor: OpenShift
+    ```
+2. A validating webhook rejects the create/update/patch requests of placements if the user has no permission on the referenced clustersets;
+3. PlacementController generates placementdecisions in the same namespace according to the spec of the placement.
+
+#### Pro
+- No need to bind clustersets to the placement namespace;
+
+#### Cons
+- The webhook can only handle create/update/patch requests of placements. If we want to use clustersets to limit the access scope of other resource, new webhook is required.
```

Comment:
it will not be easy to get the clustersets for this working ns either

Reply:
Added.